pwiz_tools/Skyline/Util/Extensions/Text.cs

```diff
@@ -456,6 +457,22 @@ public static string SpaceSeparate(params string[] values)
             return SpaceSeparate(values.AsEnumerable());
         }
 
+        /// <summary>
+        /// Append the localized colon character to a string.
+        /// </summary>
+        public static string AppendColon(string left)
+        {
+            return left + Resources.Colon;

```

Comment:
I think you should have 2 functions:

AppendLineEndingColon()
ColonSepearate()

The first is what you have for AppendColon()  but make it clear that it is for a colon going at the end of a line with no expactation that anything but a new-line will follow it.

You can't assume that appending a colon and then space separating two values is not locale specific. The Asian colon does not require a space to separate it from the following text.

Search for '：' (single character) in the zh-CHS RESX files for a better understanding of how it gets used.